baseapp_organizations/graphql/object_types.py

```
@@ -17,7 +20,9 @@ class Meta:
         )
 
     @classmethod
-    def get_node(self, info, id):
+    def get_node(
+        cls, info: graphene.ResolveInfo, id: str
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)